src/filters/dates/partialDates.test.ts

```diff
@@ -0,0 +1,143 @@
+import { DateTime } from 'luxon';
+import { TestCaseSchema } from '../../serina.schema';
+import PartialDates from './partialDates';
+
+const currentYear = DateTime.utc().year;
+
+const mockDates = (day, month, year) => DateTime.utc()
+    .set({ day, month, year })
+    .endOf('day')
+    .toJSDate();
+
+describe('Dates', () => {
+    describe('Partial Dates', () => {
+        const partialDateTestCases: TestCaseSchema[] = [{
+            case: 'go to work on 02/2009',
```

Comment:
Might want to add at least 1 test case with `-` instead of `/`?

Reply:
done

Reply:
Still can't see it